release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2733,6 +2733,40 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.15.25
+[id="ocp-4-15-25_{context}"]
+=== RHSA-2024:4955 - {product-title} 4.15.25 bug fix and security update
+
+Issued: 2024-08-07
```

Comment:
Just a heads up that Courtney seems to be changing the date format in 4.13 RNs: https://github.com/openshift/openshift-docs/pull/80085

Up to you whether you want to change the format here, or change it more comprehensively, or leave it alone to save changes for a broader sweep down the line.

Reply:
Thanks. Yes, that work is in progress. We;ll do the date updates as part of another PR.